### PR TITLE
Proxy 7-9

### PR DESCRIPTION
**Pull request by mariaYurcheuskaya**
PROXE-7: manupalating with data
PROXY-8: refactor the structure of application
PROXY_9: query params

Also added error handling middleware

### CODE REVIEW

File: src/services/meteors-service.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+const axios = require("axios");
```
Comment (plutsenka) [inline src/services/meteors-service.js:1]: Check please all usages of `''` and `""` - it's better not to mix them up

**Review by plutsenka — APPROVED** [review]

File: src/services/meteors-service.js (hunk 1 above)
Comment (m-bondarev) [reply src/services/meteors-service.js:1]: prettier can help with this
Just add some lines to `.prettierrc.json`

```
  "singleQuote": true
```

**Review by m-bondarev — APPROVED** [review]

File: src/services/meteors-service.js (hunk 1 above)
Comment (plutsenka) [reply src/services/meteors-service.js:1]: There is no such file on this repo :)

Comment (mariaYurcheuskaya) [reply src/services/meteors-service.js:1]: yep, I will add prettier to my project, Thank you!

**Pull request merged**